modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByStep.testcase

```diff
@@ -0,0 +1,274 @@
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects (Last 180 Days) time period the result and filter options are displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLast180Days {
+
+		// TODO LRQA-67684 FilterItemsFromLast180Days pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects (Last Year) time period the result and filter options displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLastYear {
+
+		// TODO LRQA-67681 FilterItemsFromLastYear pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects Today time period the result and filter options are displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromToday {
+
+		// TODO LRQA-67689 FilterItemsFromToday pending implementation
+
+	}
+
+	@description = "LPS-100679 - Verify that users are able to filter the Performance by Step page by period: Today, Yesterday, Last 7 Days, Last 30 Days, Last 90 Days, Last 180 Days, Last Year, Custom Range"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test FilterPerformanceByStepPagePerTimeRange {
+
+		// TODO LRQA-67667 FilterPerformanceByStepPagePerTimeRange pending implementation
+
+	}
+
+	@description = "LPS-98233 - Verify that the fields From and To are required"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test FromAndToAreRequiredFields {
+
+		// TODO LRQA-67686 FromAndToAreRequiredFields pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that hovering the info icon next to (Performance by Step) card will display a tooltip"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test InfoIconDisplaysMessageOnHover {
+
+		// TODO LRQA-67683 InfoIconDisplaysMessageOnHover pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that at the custom range modal, the oldest date accepted is 1970"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test OldestDataAcceptedIs1970 {
+
+		// TODO LRQA-67690 OldestDataAcceptedIs1970 pending implementation
+
+	}
+
+	@description = "LPS-100678 - Verify that the Performance by Step page displays the specific error state when there is an unexpected error"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test PageDisplaysSpecificErrorStateOnUnexpectedError {
+
+		// TODO LRQA-67669 PageDisplaysSpecificErrorStateOnUnexpectedError pending implementation
+
+	}
+
+	@description = "LPS-100677 - Verify the page is providing loading feedback when the table is being updated according to the search queries"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test PageProvidesLoadingFeedbackOnTableUpdate {
+
+		// TODO LRQA-67670 PageProvidesLoadingFeedbackOnTableUpdate pending implementation
+
+	}
+
+	@description = "LPS-100677 - Verify that the search is an approximate match (aka (LIKE) search type or (character by character comparison))"
```

Comment:
Please, remove the parenthesis from (LIKE) and (character by character comparison).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.